ANTLR/src/Compiler/SymbolTable/Action.java

```diff
@@ -0,0 +1,5 @@
+package Compiler.SymbolTable;
+
+public interface Action {
+    void execute(int i);
+}
```

Comment:
Jeg synes godt vi kunne overveje blot at overloade execute i Lambda interfacet. 
Hvad er fordele og ulemper?

Reply:
Vi kan ikke override lambda da vi skal give den en type med, da den er generic. Hvis vi forsøger at passe void klager den fordi void ikke nedarver fra Object. Hvis man i stedet giver den Void kan man godt, men det giver problemer med funktionerne. Jeg tænker at have en Action i stedet giver et godt indblik i hvad der sker her (Altså at der ikke bliver returneret noget). Måske vi i stedet skulle overveje at omdøbe lambda til function, så det også er tydeligt hvad den gør